client/src/notificationsAPI.js

```diff
@@ -0,0 +1,34 @@
+const notificationsAPI = () => {
+  return [
+    {
+      _id: "1",
+      type: "message",
+      name: "John Smith",
+      preview: "Are you going to have any vegan options in the future?",
+      link: "/chat/84ab562dfa4371243d1441",
+    },
+    {
+      _id: "2",
+      type: "message",
+      name: "Sarah Doe",
+      preview: "That sushi you made yesterday was delicious! Thank you!",
+      link: "/chat/57c86515dgf54510552",
+    },
+    {
+      _id: "3",
+      type: "order",
+      name: "Adam Sandler",
```

Comment:
He is the click handler 
![image](https://user-images.githubusercontent.com/8296126/112913070-d0b1fe80-90ad-11eb-8c7c-69b137be0acb.png)